Automattic/wp-calypso
# Sites Management Page: Use `<DropdownMenu />` core component

Fixes #66725

## Proposed Changes

Uses the `<DropdownMenu />` core component instead of Calypso's `<EllipsisMenu />`.

The icon looks a little different than the original issue because P2 uses its own ellipsis icon and I opted to save bundle size by reusing an existing `<Gridicon />`.

### After

<img width="714" alt="image" src="https://user-images.githubusercontent.com/36432/186255607-bb356fde-d260-4e63-a2c4-7bb9f3259e5b.png">


### Before

<img width="484" alt="image" src="https://user-images.githubusercontent.com/36432/186159203-2f7e4ad6-a0b7-4e8e-96c5-b333a4e654dd.png">

## Testing Instructions

1. Navigate to `/sites-dashboard`.
2. Verify the dropdown menu appears and works as expected.
